README.md

```diff
@@ -1,28 +1,27 @@
+Chassis is creating open standards designed for CSS libraries, JavaScript UI libraries, and web developers in general. This project will define standards for markup and class names for common widgets and patterns, as well as an implementation of those standards. Similar to how DOM libraries can build upon [Sizzle](https://github.com/jquery/sizzle), we hope UI libraries will build upon this project.
 
 ## Goals
 
 * **Broad Device Support**: The number of devices being used around the world continues to grow. We aim to support a wide range of devices without sacrificing usability or aesthetics.
-* **Performance**: The styles applied to your web page have a significant impact on how quickly the page can be rendered and how smoothly the page scrolls, among other things.
+* **Performance**: The styles applied to your web page have a significant impact on, for example, how quickly the page can be rendered and how smoothly the page scrolls. We aim to keep this impact as small as possible.
 * **Modularity**: Each component will be designed as separate modules, allowing customized builds of just the pieces you need.
-* **Pushing Web Standards**: We will work with W3C and WHATWG to improve CSS standards based on our work and feedback from the community.
+* **Pushing Web Standards**: We will work with the W3C and WHATWG to improve CSS standards based on our work and feedback from the community.
 * **Collaboration**: We value input and collaboration from the community. We don't want to just build yet another CSS library; we want to improve collaboration and interoperability across projects.
 
 ## Project Collaboration and Interoperability
 
-If you maintain a JavaScript library that has UI components, we'd love for you to join our efforts. We'd be happy to hear your feedback and work through updates of your project(s) to support CSS Framework. If you have the time and interest to help define the standards and improve on our implementation, that's even better! Check out the [contribution guidelines](https://github.com/jquery/css-chassis/blob/master/CONTRIBUTING.md) for more information.
+If you maintain a JavaScript library that has UI components, we'd love for you to join our efforts. We'd be happy to hear your feedback and work through updates of your project(s) to support Chassis. If you have the time and interest to help define the standards and improve on our implementation, that's even better! Check out the [contribution guidelines](https://github.com/jquery/css-chassis/blob/master/CONTRIBUTING.md) for more information.
 
 
 ## Getting started
 
 ### Prerequisites
 
-Node.js and npm should be installed. [[installer]](http://nodejs.org/download/)
+Node.js and npm [should be installed](http://nodejs.org/download/).
```

Comment:
I can't update this until next week, feel free to land this and add any remaining enhancements as needed.